flight/Modules/Stabilization/stabilization.c

```diff
@@ -123,32 +124,30 @@ static struct pios_queue *queue;
 
 uint16_t ident_wiggle_points;
 
-float axis_lock_accum[MAX_AXES] = {0,0,0};
-uint8_t max_axis_lock = 0;
-uint8_t max_axislock_rate = 0;
-float weak_leveling_kp = 0;
-uint8_t weak_leveling_max = 0;
-bool lowThrottleZeroIntegral;
-float vbar_decay = 0.991f;
+static float axis_lock_accum[MAX_AXES] = {0,0,0};
+static uint8_t max_axis_lock = 0;
+static uint8_t max_axislock_rate = 0;
+static float weak_leveling_kp = 0;
+static uint8_t weak_leveling_max = 0;
+static bool lowThrottleZeroIntegral;
 static float max_rate_alpha = 0.8f;
+float vbar_decay = 0.991f;
```

Comment:
Note: can't be static because it's also used in vbar. https://github.com/d-ronin/dRonin/blob/ec068a930321ffefda6edf0b20a10e483e30eb3e/flight/Modules/Stabilization/virtualflybar.c#L39